tests/Unit/RequestValidatorTest.php

```diff
@@ -173,27 +173,28 @@ public function __construct($array)
         $this->array = $array;
     }
 
-    function rewind()
+    function rewind(): void
     {
-        return reset($this->array);
+        reset($this->array);
     }
 
-    function current()
+    function current(): array
     {
         return [current($this->array)];
     }
 
+    #[\ReturnTypeWillChange]
     function key()
```

Comment:
Since the return type of the interface (which is `mixed` ) is not compatible with PHP prior to version 8, the attribute is added instead of the correct type.